src/rad/core.clj

```diff
@@ -47,15 +69,20 @@
   Todos:
   * Make it front-end agnostic - now it depends on many things from the terminal front end"
   [key] (do
-          (println (str "keypress: " key))
-          (if (= :enter key) ;; replace with fn frontend/normalize-key
-            (do
-              (buffer/insert-char! @point \newline)
-              ;; move point to one line down x=0, y=y+1
-              (reset! point [0 (+ 1 (second @point))]))
+          (println (str "keypress: " key ", point: " @point))
```

Comment:
Print @point after the edit is done, because `point` is modified in the rest of this function.